Kernel/Bus/PCI/Device.h

```diff
@@ -73,12 +73,13 @@ template<typename... Parameters>
 void dmesgln_pci(Device const& device, AK::CheckedFormatString<Parameters...>&& fmt, Parameters const&... parameters)
 {
     AK::StringBuilder builder;
-    if (builder.try_append("{}: {}: "sv).is_error())
-        return;
-    if (builder.try_append(fmt.view()).is_error())
-        return;
-    AK::VariadicFormatParams<AK::AllowDebugOnlyFormatters::Yes, StringView, Address, Parameters...> variadic_format_params { device.device_name(), device.device_identifier().address(), parameters... };
-    vdmesgln(builder.string_view(), variadic_format_params);
+
+    builder.appendff("{}: {}: ", device.device_name(), device.device_identifier().address());
+
+    AK::VariadicFormatParams<AK::AllowDebugOnlyFormatters::Yes, Parameters...> variadic_format_params { parameters... };
+    MUST(AK::vformat(builder, fmt.view(), variadic_format_params));
```

Comment:
Should we really MUST here? In userland, that makes sense, but the kernel should probably try and be well-behaved in low-memory situations (?)

Reply:
dbgln/dmesgln already use MUSTs, so this doesn't really change anything. But yes, we should probably deal with low-memory situations in debug output somehow (most of the time the inline capacity of the StringBuilder is enough tho).